CHANGES.rst

```diff
@@ -1,6 +1,7 @@
 2.0.1 (Unreleased)
 ==================
 - [FIX] Fixed issue with Windows platform compatibility,replaced usage of os.uname for the user-agent string.
+- [BREAKING] Moved existing query index management into new Cloudant-based design document class.
```

Comment:
I think we touched upon this on Slack but this definitely can't be part of the 2.0.1 and by the sound of things @ricellis has some reservations about it even being part of a 2.1.0 release.  In any case, you should probably remove this line item from the CHANGES.rst for now until you settle on where this work fits from a release perspective.